### PR TITLE
[Maintenance] Updating stub profile and params name

### PR DESCRIPTION
**Pull request by Mxrcon**
Hey @abhi18av following my maintenance tasks, I'm updating the stub profile with a standard container and also changing the params name as we discussed on the `dandrea_legionella` pipeline

## PR Content:
* Update to stub profile
* new params name
* splitting mtbseq into 2 modules and updating the mtbseq_per_sample output declaration

## PR ending
As always, feel free to discuss those changes with me, I'll be happy on hear your thoughts.
Hope we have a merge!

### CODE REVIEW

File: modules/mtbseq/mtbseq.nf [hunk 1]
```diff
@@ -61,12 +61,12 @@ process MTBSEQ {
...
-params.resultsDir_mtbseq_cohort = "${params.outdir}/mtbseq_cohort"
-params.saveMode_mtbseq_cohort = 'copy'
-params.shouldPublish_mtbseq_cohort = true
+params.results_dir_mtbseq_cohort = "${params.outdir}/mtbseq_cohort"
+params.save_mode_mtbseq_cohort = 'copy'
+params.should_publish_mtbseq_cohort = true
 
 process MTBSEQ_COHORT {
-    publishDir params.resultsDir_mtbseq_cohort, mode: params.saveMode_mtbseq_cohort, enabled: params.shouldPublish_mtbseq_cohort
+    publishDir params.results_dir_mtbseq_cohort, mode: params.save_mode_mtbseq_cohort, enabled: params.should_publish_mtbseq_cohort
```
Comment (abhi18av) [inline modules/mtbseq/mtbseq.nf]: Hi @Mxrcon ,

Perhaps we can also update this pipeline with the same changes as mentioned here 
https://github.com/bioinformatics-lab/mtb_mono_inh_nf/pull/8#pullrequestreview-693639958

**Review by abhi18av — CHANGES_REQUESTED** [review]
Thanks for the PR @Mxrcon , I have requested a tiny change, which is not urgent. Though it'd be nice to have that done as part of this PR.

File: modules/mtbseq/mtbseq.nf (hunk 1 above)
Comment (Mxrcon) [reply modules/mtbseq/mtbseq.nf]: yes sure! I'll implement that very soon

Comment (Mxrcon) [reply modules/mtbseq/mtbseq.nf]: @abhi18av Also: Do you think that we should update the output declaration from `mtbseq_per_sample` to follow the same pattern of `rita bahia`? or we should keep them differently.

Comment (abhi18av) [reply modules/mtbseq/mtbseq.nf]: `mtbseq_profile` ? 🤔

Comment (Mxrcon) [reply modules/mtbseq/mtbseq.nf]: ohh i was confused, its` mtbseq_per_sample`

Comment (abhi18av) [reply modules/mtbseq/mtbseq.nf]: > Do you think that we should update the output declaration from mtbseq_per_sample to follow the same pattern of rita bahia? or we should keep them differently.

Please be specific - what changes exactly?

Comment (Mxrcon) [reply modules/mtbseq/mtbseq.nf]: on rita you use:

```
    output:
    path("${genomeFileName}")
```

and on mtb_mono:

```
    output:
    val("${genomeFileName}") // Genome name
    path("""${genomeFileName}_results""") // Folder
    path("""${genomeFileName}_results/Called/*tab""")
    path("""${genomeFileName}_results/Position_Tables/*tab""")
```
Would you like change one of them to make it the same on both modules?

Comment (abhi18av) [reply modules/mtbseq/mtbseq.nf]: Absolutely, the `mtb_mono` one is the correct one 👍

Comment (abhi18av) [reply modules/mtbseq/mtbseq.nf]: But while making these changes please also make sure that the rest of the directives have the same content.

Comment (Mxrcon) [reply modules/mtbseq/mtbseq.nf]: yes, sure, I'll update it accordingly on this PR!

**Comment by Mxrcon** [comment]
@abhi18av I've updated this pipeline with the changes that you requested, so i gently request another review! I've made a small tweak on the main.nf, please check cd8fc96da7ca487b99552afd5add78bcede9c824.

**Review by abhi18av — APPROVED** [review]
Looks good to merge ✅

**Review by abhi18av — APPROVED** [review]
Looks good to merge ✅ 

Thanks @Mxrcon !

**Comment by Mxrcon** [comment]
@abhi18av I think that there was a problem with this merge and the other ones :fearful:

**Comment by abhi18av** [comment]
@Mxrcon , please elaborate?

**Comment by Mxrcon** [comment]
> @Mxrcon , please elaborate?

It was aproved but not merged, the other prs are in the same state

**Comment by abhi18av** [comment]
Ah, I forgot to merge. Whoops 😅

**Comment by Mxrcon** [comment]
No problem :laughing: :rofl:

**Pull request merged**